src/instance.ts

```diff
@@ -193,6 +248,255 @@ class Instance extends common.GrpcServiceObject {
+   * const backup = instance.backup('my-backup');
+   */
+  backup(backupId: string): Backup {
+    if (!backupId) {
+      throw new Error('A backup ID is required to create a Backup.');
+    }
+
+    return new Backup(this, backupId);
+  }
+
+  getBackups(options?: GetBackupsOptions): Promise<GetBackupsResponse>;
+  getBackups(callback: GetBackupsCallback): void;
+  getBackups(query: GetBackupsOptions, callback: GetBackupsCallback): void;
+  /**
+   * Query object for listing backups.
+   *
+   * @typedef {object} GetBackupsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupsResponse
+   * @property {Backup[]} 0 Array of {@link Backup} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List backups on the instance.
+   *
+   * Both completed and in-progress backups are listed if no filter is supplied.
+   *
+   * @see {@link #backup}
+   *
+   * @param {GetBackupsOptions} [options] Contains query object for listing
+   *     backups and request configuration options, outlined here:
+   *     https://googleapis.github.io/gax-nodejs/CallSettings.html.
+   * @returns {Promise<GetBackupsResponse>} when resolved, contains a paged list of backups.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [backups] = await instance.getBackups();
+   */
+  getBackups(
+    optionsOrCallback?: GetBackupsOptions | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const callback =
+      typeof optionsOrCallback === 'function'
+        ? (optionsOrCallback as GetBackupsCallback)
+        : cb!;
+    const options =
+      typeof optionsOrCallback === 'object'
+        ? (optionsOrCallback as GetBackupsOptions)
+        : {gaxOptions: {}};
+    const gaxOpts: CallOptions = options.gaxOptions as CallOptions;
+    const reqOpts = extend({}, options, {
+      parent: this.formattedName_,
+    });
+    delete reqOpts.gaxOptions;
+
+    this.request<IBackup[]>(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts,
+      },
+      (err, rowBackups, ...args) => {
+        let backups: Backup[] | null = null;
+        if (rowBackups) {
+          backups = rowBackups.map(rowBackup => {
+            return this.backup(rowBackup.name!);
+          });
+        }
+
+        callback(err, backups, ...args);
+      }
```

Comment:
@alexander-fenster regarding the failing system and samples tests, I think it does have something to do with types but I'm not sure exactly what has changed. `getBackups()` is setup in an identical way to `getDatabases()`.

https://github.com/googleapis/nodejs-spanner/blob/ccf1b61bcf060a72c35712c5d0e529fad9684724/src/instance.ts#L694-L711

Yet when you call the RPCs in the system tests, the second arg of the `getDatabases()` callback is `IDatabase[]` as requested but the second arg of `getBackups()` is of type `IListBackupsResponse`. This is why the `rowBackups.map()` call is not there since what's returned is not a list.

Any pointers you have for debugging this is appreciated.